apps/console/src/features/applications/components/forms/inbound-oidc-form.tsx

```diff
@@ -259,6 +262,16 @@ export const InboundOIDCForm: FunctionComponent<InboundOIDCFormPropsInterface> =
         }
     }, [ template ]);
 
+    useEffect(() => {
```

Comment:
Add a doc comment explaining this hook.

Reply:
fixed with b02cac6a564ad16853cc79e541e354bc911f9a7b